### PR TITLE
[doc] credential plugins documentation: mention setup_managed_credential_types command & fix entry point example

### PR DESCRIPTION
**Pull request by pilou-**
##### SUMMARY
Credential plugins documentation:
* mention `setup_managed_credential_types` command
* fix `awx.credential_plugins`  entry point example

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME

- docs

### CODE REVIEW

**Comment by AlanCoding** [comment]
@ryanpetrello @jakemcdermott is this true?

File: docs/credentials/credential_plugins.md [hunk 1]
```diff
@@ -117,12 +117,22 @@ setuptools.setup(
...
+            'fancy = my_fancy_project.my_fancy_module:some_fancy_plugin',  # some_fancy_plugin = CredentialPlugin(...)
         ]
     }
 )
 ```
 
+Enable Custom Credential Plugins
+--------------------------------
+
+Once the Python package has been installed, `awx-manage` must be used in order
```
Comment (ryanpetrello) [inline docs/credentials/credential_plugins.md:129]: What we've *been* doing is introducing a migration with each _new_ plugin definition that *calls* this function and auto-detects and registers it.  That's probably the more upgrade-safe approach.

Comment (pilou-) [reply docs/credentials/credential_plugins.md:129]: That doesn't apply to custom credential plugins, isn't it ?

**Comment by ryanpetrello** [comment]
@pilou- sorry it took me so long to follow up on this.  Here's a replacement for this PR that I think is a better alternative:

https://github.com/ansible/awx/pull/6717

**Pull request closed without merging**